flink-libraries/flink-table/src/test/scala/org/apache/flink/table/runtime/stream/sql/OverWindowITCase.scala

```diff
@@ -50,6 +50,80 @@ class OverWindowITCase extends StreamingWithStateTestBase {
     (8L, 8, "Hello World"),
     (20L, 20, "Hello World"))
 
+  @Test
+  def testUnboundedDistinctGroupWindow(): Unit = {
```

Comment:
Remove this test case. The distinct in this case is solved by `AggregateExpandDistinctAggregatesRule`

Reply:
oops.. this was suppose to be added later in FLINK-8690 :-P